packages/logout-extension/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "jupyterlab-logout-extension",
```

Comment:
Should the `npm` name keep the `-extension` suffix? Or just be `jupyterlab-logout` like the current name on npm? https://www.npmjs.com/package/jupyterlab-logout

Reply:
Just so that when we make a release later with this change we still publish to the same package on npm.